transifex/native/parsing.py

```diff
@@ -67,6 +75,14 @@ def tags(self):
         """
         return self.meta.get(consts.KEY_TAGS, [])
 
+    def add_occurrence(self, occurrence):
+        """An optional occurrence field, shows were the string is located
+
+        :param unicode occurrence: <file_path>:<line_number>, indicates the
+            exact location of the occurrence
+        """
+        self.occurrence += ";" + occurrence
```

Comment:
right now, `.occurrence` is a read-only property. Doing a `+=` on it will not work. You need to add a:

```python
@occurrence.setter
def occurrence(self, value):
    self.meta[const.KEY_OCCURENCE] = value
```

above

Reply:
yep that's right! I forgot that one dec!